website/search/file_util.py

```diff
@@ -0,0 +1,74 @@
+import requests
+
+from framework.exceptions import HTTPError
+
+
+INDEXED_TYPES = [
+    'txt',
+    'md',
+    'rtf',
+    'docx',
+    'pdf',
+]
+
+
+def is_indexed(filename):
+    extension = filename.rsplit('.')[-1]
+    indexed = extension in INDEXED_TYPES
+    return indexed
+
+
+def get_content_of_file(file_):
```

Comment:
I can't think of anywhere else we use "of" in a function name. Consider renaming to `get_file_content`.